sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -5006,7 +5015,8 @@
                             "properties": {},
                             "allowMultipleMasks": false,
                             "addons": [],
-                            "tag": "p"
+                            "tag": "p",
+                            "className": ""
```

Comment:
just curious, why this `className` is getting added to all p tags.

Reply:
This is the first time we are extracting form defintions for noticeofassessment through form.io.

May be that is the difference between form.io and formsflow.ai